harness/determined/core/_context.py

```diff
@@ -123,6 +124,9 @@ def init(
             ``core_context.preempt.should_preempt()`` method.  See
             :class:`~determined.core.PreemptMode` for more detail.  Defaults to ``WorkersAskChief``.
         storage_manager: Internal use only.
+        tensorboard_mode (``core.TensorboardMode``, optional): Define how Tensorboard
+            metrics and profiling data are retained. See
+            :class:`~determined.core.TensorboardMode`` for more detail.
```

Comment:
Nit:  include a `Defaults to` sentence at the end, according to [this guide](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)